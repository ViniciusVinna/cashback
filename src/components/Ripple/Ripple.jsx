import React from 'react';
import Ink from 'react-ink';

const config = {
  background: true,
  duration: 1000,
  opacity: 0.05,
  radius: 150,
};

const Ripple = () => (<Ink {...config} data-testid="ripple" />);

export default Ripple;
