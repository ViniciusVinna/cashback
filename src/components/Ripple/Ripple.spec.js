import React from 'react';
import Ripple from './Ripple';

describe('Ripple', () => {
  it('should render properly', async () => {
    const { container } = await render(<Ripple />);

    expect(container).toMatchSnapshot();
  });
});
