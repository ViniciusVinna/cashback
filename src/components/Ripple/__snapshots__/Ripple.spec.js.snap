// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ripple should render properly 1`] = `
<div>
  <canvas
    height="0"
    style="border-radius: inherit; height: 100%; left: 0px; position: absolute; top: 0px; width: 100%;"
    width="0"
  />
</div>
`;
